### DISCUSSION TITLE
1.12 Mod version

### DISCUSSION THREAD

**Discussion by solundev** [HbmMods/Hbm-s-Nuclear-Tech-GIT#520] [Q&A] [answered]
So, does the 1.12 version have everything okay as 1.7.10 ?
because i'm looking forward for upgrading the version for new maps but i don't wan't to mess the experience i'm having with the mod in actual version, so would be nice to know more about

**Comment by SoB1nary** [comment, marked as answer]
short answer: no

**Reply by solundev** [reply]
thanks, that's sad because i love this mod it's a unique one, but it's really sad cuz at 1.12.2 it crashes the game while looking at inv on creative switching tabs, and even with JEI...
